Update sit-limits.md (#12119)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/sit-limits.md

|Maximum size of a keyword dictionary (post compression)| 1MB (~1,000,000 characters)|
|Maximum number of keyword dictionary based SITs in a tenant| 50 |
|Maximum number of MIP+MIG policies in a tenant| 10,000 |

> |Maximum number of DLP rules in a policy | upto 600, as long as the policy size remains below 100KB and there are a maximum of 600 rules in the tenant |

Reviewer: ```suggestion
|Maximum number of DLP rules in a policy | up to 600, as long as the policy size remains below 100KB and there is a maximum of 600 rules in the tenant |
```
